client/my-sites/customer-home/main.jsx

```diff
@@ -310,7 +310,7 @@ class Home extends Component {
 									external
 									onClick={ () => trackAction( 'support', 'docs' ) }
 								>
-									{ translate( 'Support docs' ) }
+									{ translate( 'Support articles' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'All support articles' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/customer-home-support-link)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).